conf/modules/gpio_ext_pca95xx.xml

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+<module name="gpio_ext_pca95xx" dir="gpio_ext" task="mcu">
+  <doc>
+    <description>
+PCA95XX external GPIO peripheral
+
+Uses the gpio_ext_common mechanism for transparent redirecting of gpio
+calls. Multiple PCA95XX can be used, on different I2C busses.
+Configuration and writing can be performed non-blocking (reading is
+always blocking).
+    </description>
+    <section name="GPIO_EXT" prefix="GPIO_EXT_">
+      <define name="PROVIDERx" value="GPIO_EXT_PCA95XX" description="Set-up module as provider for port GPIOEXTx"/>
+      <define name="BLOCKINGx" value="TRUE|FALSE" description="Whether write operations to this port should be blocking. (Note: reading is always blocking)."/>
+      <define name="I2C_PERIPHx" value="&i2c0" description="Pointer(!) to i2c peripheral"/>
```

Comment:
to avoid using `&` for consistency with other modules, you can have in your code:
```
#ifdef I2C_PERIPHx
#define I2C_PERIHx_PTR &(I2C_PERIPHx)
#else
#define I2C_PERIHx_PTR NULL
#endif
```

Reply:
Thanks for the suggestion, fixed. I've also updated the module xml to generate the missing USE_I2Cx defines.